internal/contracts/registry.go

```diff
@@ -1958,6 +2190,27 @@ func (_Registry *RegistryTransactorSession) MintVehicleAndSdWithDeviceDefinition
 	return _Registry.Contract.MintVehicleAndSdWithDeviceDefinitionSign(&_Registry.TransactOpts, data)
 }
 
+// MintVehicleAndSdWithDeviceDefinitionSignWithSacd is a paid mutator transaction binding the contract method 0x62a147ff.
```

Comment:
I think this needs to be regenerated with the new name mintVehicleAndSdWithDeviceDefinitionSignAndSacd